evoting/protocol/decrypt_test.go

```diff
@@ -72,11 +73,10 @@ func runDecrypt(t *testing.T, n int) {
 
 	select {
 	case <-decrypt.Finished:
-		// partials, _ := election.Partials()
-		// for _, partial := range partials {
-		// 	fmt.Println(partial)
-		// 	assert.False(t, partial.Flag)
-		// }
+		partials, _ := election.Partials()
+		for _, partial := range partials {
+			log.Lvl1(partial)
+		}
```

Comment:
no assertion here?

Reply:
I only uncommented this from Andrea's code - the assertion there was failing. So I thought that I might at least replace the `fmt.Pringln` with a `log.Lvl1`...